cri/v1alpha1/cri_wrapper.go

```diff
@@ -100,15 +106,17 @@ func (c *CriWrapper) PodSandboxStatus(ctx context.Context, r *runtime.PodSandbox
 
 // ListPodSandbox returns a list of Sandbox.
 func (c *CriWrapper) ListPodSandbox(ctx context.Context, r *runtime.ListPodSandboxRequest) (res *runtime.ListPodSandboxResponse, err error) {
-	logrus.Infof("ListPodSandbox with filter %+v", r.GetFilter())
-	defer func() {
-		if err != nil {
-			logrus.Errorf("failed to ListPodSandbox: %v", err)
-		} else {
-			// NOTE: maybe log detailed sandbox items with higher log level.
-			logrus.Infof("success to ListPodSandbox: %+v", res.Items)
-		}
-	}()
+	if c.debug {
```

Comment:
I do not think add `c.debug` is a good way. How about make `logrus.Infof` to be `logrus.Debugf`?